src/ProductFilters/WidgetFilterDeliveryTime.php

```diff
@@ -154,7 +154,7 @@ public static function getProductsDeliveryTimes(array $args = []) {
    *
    * @implements get_terms_orderby
    */
-  public static function get_terms_orderby(string $orderby, array $query_vars, array $taxonomy = []): string {
+  public static function get_terms_orderby(string $orderby, array $query_vars, ?array $taxonomy): string {
```

Comment:
From https://www.php.net/manual/en/migration71.new-features.php:

The nullable type declaration only works if the calling code explicitly passes `NULL`, otherwise it throws:
> Uncaught Error: Too few arguments to function test(), 2 passed in...

Is this your intention?

```suggestion
  public static function get_terms_orderby(string $orderby, array $query_vars, array $taxonomy = []): string {
```